gtp.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Google LLC
```

Comment:
should probably have used `git mv` to preserve history?

Reply:
i'll change history just for you :P

Reply:
no but actually, I blame git diff

https://github.com/tensorflow/minigo/pull/341/commits/1a6f8fdd3dd1db6d15dcca9b4660297e8730c51e